nncf/experimental/tensorflow/graph/transformations/layout.py

```diff
@@ -0,0 +1,45 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from nncf.common.graph.transformations.commands import TransformationCommand
+from nncf.common.graph.transformations.commands import TransformationType
+from nncf.common.graph.transformations.layout import TransformationLayout
+
+
+class TFTransformationLayoutV2(TransformationLayout):
```

Comment:
Same comment about V2

Reply:
Thanks for the comment! I added the additional explanation to the docstring.